magento/magento2
# M2C-21768 Validate product quantity on Wishlist update

### Description (*)

I've added a method which validates product qty in order to prevent updating Wishlist with product quantity that is empty, 0 or negative number.

### Fixed Issues (if relevant)

1. magento/magento2#25032: Display some error "We can't update your Wish List right now." at wish list

### Manual testing scenarios (*)

1. Add product to Wishlist
2. Go to Wishlist
3. Edit the product from Wishlist
4. On detail page leave qty blank, add 0 or negative number and click on Update Wishlist button.
5. Error message should appear.

![image](https://user-images.githubusercontent.com/34317830/69068377-ef69a500-0a24-11ea-9944-cf1c26c161f9.png)

### Questions or comments

The same behaviour works for updating products quantity in Wishlist in User Area. 

![image](https://user-images.githubusercontent.com/34317830/69068608-5b4c0d80-0a25-11ea-88c7-1039a5fb70aa.png)

### Contribution checklist (*)
